src/components/main-navigation/template.html

```diff
@@ -14,7 +14,7 @@
       <content select="secondary-items" id="secondary-items"></content>
 
       <template is="dom-repeat" items="{{secondaryItems}}">
-        <nav-item text="{{item.text}}" template="{{item.template}}" location="{{item.location}}" active="{{item.active}}" items="{{item.items}}" props="{{item.props}}" attrs="{{item.attrs}}" classes="{{item.classes}}" class$="dropdown {{dashed(item.text)}}" dropdown="{{item.items.length}}"></nav-item>
+        <nav-item item="{{item}}" class$="dropdown {{dashed(item.text)}}" dropdown="{{item.items.length}}"></nav-item>
```

Comment:
What happen if I set dropdown to false, but it still has items ? Should it be item.dropdown then? Example use case if I don't want to display the dropdown now, but I want to keep the items there.

Reply:
Well nav-item support the dropdown prop but in this scenario its more that we loop all items in the secondary-item part and per default display their childs in a dropdown.
If you want to override the for some reason if might work to set props.dropdown = true but we could look at that scenario later on if we think that is important.